### PR TITLE
(GH-29) Error if template not found

### PR DESCRIPTION
**Pull request by jpogran**
Add a new func to the `pct` pacakge to retrieve a PCT from the local filesystem. Modify `pct new` to return a formatted error if the PCT is not found.

### CODE REVIEW

File: internal/pkg/pct/pct_test.go [hunk 1]
```diff
@@ -69,6 +69,52 @@ func TestDeploy(t *testing.T) {
...
+		selectedTemplate string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    PuppetContentTemplate
+		wantErr bool
+	}{
+		{
+			name:    "returns error for non-existant template",
```
Comment (sanfrancrisko) [inline internal/pkg/pct/pct_test.go:84]: nit:
```suggestion
			name:    "returns error for non-existent template",
```

File: internal/pkg/pct/pct_test.go [hunk 2]
```diff
@@ -69,6 +69,52 @@ func TestDeploy(t *testing.T) {
...
+			args:    args{},
+			wantErr: true,
+		},
+		{
+			name: "returns tmpl for existant template",
```
Comment (sanfrancrisko) [inline internal/pkg/pct/pct_test.go:89]: nit:
```suggestion
			name: "returns tmpl for existent template",
```

File: cmd/new/new.go [hunk 3]
```diff
@@ -133,6 +133,11 @@ func execute(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
+	_, err := pct.Get(localTemplateCache, selectedTemplate)
+	if err != nil {
+		log.Fatal().Msgf("Couldn't find an installed template that matches '%s'", selectedTemplate)
```
Comment (sanfrancrisko) [inline cmd/new/new.go:138]: Could we use the existing error message like [this](https://golang.org/pkg/fmt/#example_Errorf)? 
```suggestion
		log.Fatal().Msgf(err.Error)
```

**Review by sanfrancrisko — CHANGES_REQUESTED** [review]

File: cmd/new/new.go (hunk 3 above)
Comment (jpogran) [reply cmd/new/new.go:138]: Good catch! I moved the custom error message to the pct get method but forgot to remove it from the new command.

**Review by sanfrancrisko — APPROVED** [review]

**Pull request merged**